Userland/Libraries/LibGfx/ImageFormats/JPEGLoader.cpp

```diff
@@ -1082,11 +1080,14 @@ static ErrorOr<void> read_quantization_table(Stream& stream, JPEGLoadingContext&
             return Error::from_string_literal("Unsupported unit hint in quantization table");
         }
         u8 const table_id = info_byte & 0x0F;
-        if (table_id > 1) {
+
+        if (table_id > 3) {
```

Comment:
Should we keep rejecting table_id > 1 for baseline jpegs? Those are only allowed to use 2 tables.

Reply:
From `B.2.4.1 - Quantization table-specification syntax`:

![table](https://user-images.githubusercontent.com/26030965/233796798-b85d04c5-86eb-4da0-be1b-050030e01b89.png)


So it looks like even baseline DCT can have more quantization tables and btw, both faulty images are SOF0